lib/oembed/providers/builtin_providers.rb

```diff
@@ -6,6 +6,7 @@
 require 'oembed/providers/facebook_video'
 require 'oembed/providers/instagram'
 require 'oembed/providers/matterport'
+require 'oembed/providers/spotify'
```

Comment:
**Annotation:** I've been moving more & more of the built-in providers to their own files, for better organization & fuzzy-file-name findablility.